ProjectShop/ProductApp/models.py

```diff
@@ -117,78 +128,110 @@ def __str__(self):
         return self.name
 
     @property
     def name(self):
-        return f'{self.user} reply on {self.review} with reaction: {self.reaction}'
+        return f"{self.user} reply on {self.review} with reaction: {self.reaction}"
 
 
-class ProductMedia(models.Model):
+class ProductImage(models.Model):
     """
-    A database object that represents media bound to the product, like pictures or video links.
+    A database object that represents images to the product.
 
     Attributes:
-        media_type: indicates whether media is a picture or a video iframe.
         image: reference to image saved in django 'media' file.
-        video_link: link to data that iframe is using.
         product: reference to target product.
     """
 
-    MEDIA_TYPES = [
-        (0, 'picture'),
-        (1, 'video_link'),
-    ]
-
-    product = models.ForeignKey(Product, blank=False, null=False, on_delete=models.PROTECT, related_name='media')
+    product = models.ForeignKey(
+        Product, blank=False, null=False, on_delete=models.PROTECT, related_name="image"
+    )
 
-    media_type = models.PositiveSmallIntegerField(choices=MEDIA_TYPES, null=False, blank=False)
-    image = models.ImageField(upload_to="product_media_image", default='default_image/default_image.png')
-    video_link = models.URLField(null=True, blank=True)
+    image = models.ImageField(
+        upload_to="product_media_image", default="default_image/default_image.png"
+    )
 
     class Meta:
-        verbose_name_plural = _('Product media')
+        verbose_name_plural = _("Product media")
 
     def small_image_tag(self):
-        return format_html('<img href="{0}" src="{0}" width="50" height="50" />'.format(self.image.url))
+        return format_html(
+            '<img href="{0}" src="{0}" width="50" height="50" />'.format(self.image.url)
+        )
 
     small_image_tag.allow_tags = True
-    small_image_tag.short_description = _('Image')
+    small_image_tag.short_description = _("Image")
 
     def image_tag(self):
-        return format_html('<img href="{0}" src="{0}" width="150" height="150" />'.format(self.image.url))
+        return format_html(
+            '<img href="{0}" src="{0}" width="150" height="150" />'.format(
+                self.image.url
+            )
+        )
 
     image_tag.allow_tags = True
-    image_tag.short_description = _('Image')
+    image_tag.short_description = _("Image")
+
+    def __str__(self):
+        return self.name
+
+    @property
+    def name(self):
+        return f"Image {self.id} of {self.product}"
+
+
+class ProductVideo(models.Model):
+    """
+    A database object that represents video link to the product.
+
+    Attributes:
+        video_link: link to data that iframe is using.
+        product: reference to target product.
+    """
+
+    product = models.ForeignKey(
+        Product,
+        blank=False,
+        null=False,
+        on_delete=models.PROTECT,
+        related_name="video_link",
+    )
```

Comment:
The default value for blank and null is False, so there is no need to specify these arguments explicitly.
For more information, please, visit the link:
https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.Field.null

Reply:
well, it was not written by me, but I will fix this